docs/json-api.md

```diff
@@ -34,6 +34,16 @@ const styles = {
 }
 ```
 
+To use the '!important' modifier with function values, you must use [array syntax](./js-api.md#alternative-syntax-for-space-and-comma-separated-values):
+
+```
+const styles = {
+  button: {
+    color: data => [[data.color, '!important']]
```

Comment:
I am asking myself what more correct:

`[[data.color, '!important']]` vs. `[data.color, '!important']` and `[[data.color],  '!important']`

Reply:
On one hand, space separated values are separated using array syntax using double arrays. On the other hand !important is not really part of the value itself, it is more like a flag.

WDYT?

Reply:
I like the idea of it just being a single array: [data.color, '!important'], and that's how I originally had the code structured, but it adds the comma in the text when calling the 'toCssValue' method.  I guess since we're already doing a substring to remove the '!important' from the text, we could just make it remove the comma in front of it as well if we wanted to keep it as a single array. Thoughts?

Reply:
We could teach toCssValue handle last element properly for e.g., not sure if it is better.

Reply:
I guess I would lean on your knowledge to decide what is best for the overall architecture. Either way I'm hoping we can release these soon :).  Let me know if you'd like me to change anything.

Reply:
The thing is if we don't teach toCssValue to correctly stringify a value with !important, the function toCssValue itself won't be correct, because it will convert  `toCssValue(['red', '!important'])` to `'red, !important'`.  So I think we should change it.

Reply:
That being said, I think perhaps to keep things simple and not add overhead to the 'toCssValue' function we should just leave it as it stands using the double array syntax:  [['red', '!important']].  Modifying the 'toCssValue' method just to add logic for the "!important" modifier might degrade performance.  I'm not sure where and how often it's called and so I would leave that to your judgement. Thoughts?

Reply:
Need to check what needs to be done in toCssValue for this